### PR TITLE
feat: move all linking targets to /usr/local/bin

### PR DESCRIPTION
**Pull request by Chumper**
Moves the linking path to `/usr/local/bin`

### CODE REVIEW

**Review by viceice — COMMENTED** [review]
needs some test fixes 😏

File: test/bash/linking.bats [hunk 1]
```diff
@@ -39,22 +41,22 @@ teardown() {
 
   run link_wrapper git
   assert_success
-  assert [ -f "${USER_HOME}/bin/git" ]
+  assert [ -f "${BIN_DIR}/git" ]
```
Comment (viceice) [inline test/bash/linking.bats:44]: So git is just a random test executable?

**Review by viceice — APPROVED** [review]

File: test/bash/linking.bats (hunk 1 above)
Comment (Chumper) [reply test/bash/linking.bats:44]: yes, I can use a more standard executable like ls, cat or sort like that

Comment (viceice) [reply test/bash/linking.bats:44]: Leave as is. 🤗

**Pull request merged**